pypeman/commands.py

```diff
@@ -71,8 +105,23 @@ def main(debug_asyncio=False, profile=False, cli=False, remote_admin=False):
     load_project()
     print('\nStarting...')
 
+    logger = logging.getLogger(__name__)
     loop = asyncio.get_event_loop()
 
+    ctx = dict(
+        logger=logger,
+        loop=loop,
+        )
+
+    signal_handler = partial(sig_handler_func, ctx=ctx)
+
+    signal.signal(signal.SIGINT, signal_handler)  # CTRL-C
```

Comment:
May be you should use https://docs.python.org/3/library/asyncio-eventloop.html#asyncio.loop.add_signal_handler instead of sync signal ? It seems easier to interact with loop this way.

Reply:
have to check whether there was any particular reason I didn't use this.
I'm aware about the function.

Will either switch over or explain why I don't use it.

Reply:
Don't know anymore why I didn't loop.add_signal_handler()

Fixed in commit 8abca7aad7c52770fbd1a5d617aa27cd310ee027